lib/Doctrine/Migrations/Configuration/Migration/YamlFile.php

```diff
@@ -35,7 +35,7 @@ public function getConfiguration(): Configuration
 
         try {
             $config = Yaml::parse($content);
-        } catch (ParseException $e) {
+        } catch (ParseException) {
```

Comment:
shouldn't we rather attach it as a previous exception, so that the user can see the actual parsing issue ?

Reply:
Yes, but that's out if scope for this PR. We can do this in a follow-up.